guide/portuguese/electron/index.md

```diff
@@ -23,8 +23,11 @@ Alguns aplicativos criados usando o Electron incluem:
 
 ### Referências adicionais de informação
 
+https://electronjs.org/apps
+
 *   [Site oficial](https://electronjs.org/)
+*   [Lista de Aplicações em Electron](https://electronjs.org/apps)
```

Comment:
This is redundant with the main website just over it

Reply:
Are you suggesting to remove line 26 and 29?

Reply:
Only line 29

Reply:
I will remove both of them.